shoop/front/middleware.py

```diff
@@ -74,7 +76,7 @@ def _set_shop(self, request):
             raise ImproperlyConfigured("No shop!")
 
     def _set_person(self, request):
-        request.person = get_person_contact(request.user)
+        request.person = get_company_contact(request.user) or get_person_contact(request.user)
```

Comment:
I think this should be like it was before. request.person can't be company contact.